src/app/OperationalSessionSetup.cpp

```diff
@@ -190,6 +191,17 @@ void OperationalSessionSetup::Connect(Callback::Callback<OnDeviceConnected> * on
     Connect(onConnection, onFailure, nullptr, transportPayloadCapability);
 }
 
+void OperationalSessionSetup::Connect(Callback::Callback<OnDeviceConnected> * onConnection, Callback::Callback<OnDeviceConnectionFailure> * onFailure, 
+                                      RendezvousParameters & params,
+                                      TransportPayloadCapability transportPayloadCapability)
+{
+    mState = State::ResolvingAddress;
+    mTransportPayloadCapability = transportPayloadCapability;
+
+    UpdateDeviceData(params.GetPeerAddress(), params.GetMRPConfig());
```

Comment:
So just trying to understand this: we are establishing CASE here, right?  For a device that is presumably not on-network?  So are we doing CASE over some non-IP transport in this case?

We should put some thought into how to make this zero-cost or close to it for nodes that never perform network recovery but do need CASE establishment in general (e.g. for bindings).